### DISCUSSION TITLE
Alterar o algoritmo de busca para priorizar título

### DISCUSSION THREAD

**Discussion by george-gca** [levxyca/diciotech#192] [Ideas]
Atualmente se fizermos a busca por `algoritmo` no diciotech, a busca dá certo, mas o card de `Algoritmo` é a 2a opção nos resultados. Pode ser que em algum caso o resultado "principal" apareça ainda mais pra baixo nos resultados. Acho que seria interessante priorizar na lista de resultados os cards que possuem a palavra no título para depois mostrar resultados onde aparecem só na explicação.

EDIT: agora que vi que ele na verdade só esconde os itens que não passam no filtro da busca, então ele não faz nenhuma edição atualmente no html depois de criado a não ser esconder/mostrar. Pra isso acho que seria necessário alterar a árvore dom de acordo com a busca.

**Comment by levxyca** [comment]
Excelente observação @george-gca! Acredito que vale abrirmos uma issue pra isso, o que acha?

**Reply by george-gca** [reply]
Transferindo a discussão pra #196.

**Reply by levxyca** [reply]
Perfeito!
